spec/services/movie_service_spec.rb

```diff
@@ -38,6 +38,7 @@
   describe "#results_page_count" do
     it "returns an integer from JSON data regarding number of pages of results" do
         movie_service = MovieService.new
+        # Where are you setting this env var locally?
```

Comment:
I set that variable locally in application.yaml along with the API_KEY var. This is a work around I used to feed a url directly into the test on line 8 without exposing my token. There has to be a better solution but thats the one I came up with yesterday. I'm wondering if the 'get_data' method is inherently tested by all the other methods functioning and passing tests.

Reply:
And Travis CI has this ENV variable so it should pass on the build